package.json

```diff
@@ -69,6 +69,7 @@
     "@babel/core": "7.11.5",
     "@cozy/codemods": "1.2.0",
     "@material-ui/core": "3.9.4",
+    "@material-ui/styles": "4.10.0",
```

Comment:
Je vois rien qui se base sur ce paquet, pourquoi on a t on besoin ?

Reply:
C'est Box qui s'appuie là-dessus. Ok merci pour @material-ui/core/styles. Peut-être forcer la version à la dernière v3 également ?

Reply:
https://github.com/mui-org/material-ui/blob/v3.x/packages/material-ui/src/Box/Box.js#L5 donc c'est pas la même chose merci 👍 

OK, ouais je pense que c'est mieux de rester en v3 je pense

Reply:
Hmmm la version de ce paquet est indépendante de la version du core je pense https://v3.material-ui.com/customization/css-in-js/ non ? La version 3 est vielle de 2 ans et en alpha https://www.npmjs.com/package/@material-ui/styles?activeTab=versions . Il vaut peut-être mieux rester sur la dernière stable ? On aura de plus peut-être moins de souci en passant à MUIv4 étant donné que là ce n'est utilisé que dans le cas de Box, et que ça fonctionne ?